build/assets/themes/dashboard/scss/main.scss

```diff
@@ -39,45 +42,56 @@ div#ccm-dashboard-content {
       font-weight: 200;
       margin-top: 0px;
       line-height: inherit;
-      color: #333;
-      border-bottom: 1px solid #e5e5e5;
+      color: $gray-700;
+      border-bottom: 1px solid $gray-200;
     }
   }
 
   position: relative;
-  padding-right: 50px;
-  padding-left: 97px;
-  padding-bottom: 100px;
+
+  padding: 48px 50px 100px 45px;
 
   > header {
-    margin-left: -97px;
-    margin-right: -50px;
-    padding: 39px 80px 39px 97px;
+    padding: 40px 0 20px 0;
     position: relative;
 
+    .header-icon {
+      .icon-bookmark {
+        height: 40px;
+        width: 40px;
+        margin-top: -10px;
+        //SVG cant render $gray-700 variable
+        --icon-color: #4C4F56;
```

Comment:
Does it work to use interpolation here: https://sass-lang.com/documentation/interpolation
it'd be: `#{$gray-700}`

Reply:
this worked like a charm! thanks @KorvinSzanto